### ISSUE TITLE
SVGP q_sqrt can't be fixed

### ISSUE THREAD

**Issue by msbauer** [GPflow/GPflow#46]
When fixing q_sqrt in SVGP i cannot compile the model as it breaks with the following error message
```
Traceback (most recent call last):
  File "/path/Debug.py", line 24, in <module>
    m._compile()
  File "/path/GPflow/GPflow/svgp.py", line 81, in _compile
    opt_step = GPModel._compile(self, optimizer)
  File "/path/GPflow/GPflow/model.py", line 154, in _compile
    f = self.build_likelihood() + self.build_prior()
  File "/path/GPflow/GPflow/svgp.py", line 112, in build_likelihood
    fmean, fvar = conditionals.gaussian_gp_predict_whitened(self._tfX, self.Z, self.kern, self.q_mu, self.q_sqrt, self.num_latent)
  File "/path/GPflow/GPflow/conditionals.py", line 100, in gaussian_gp_predict_whitened
    return conditional(Xnew, X, kern, q_mu, num_columns=num_columns, full_cov=full_cov, q_sqrt=q_sqrt, whiten=True)
  File "/path/GPflow/GPflow/conditionals.py", line 72, in conditional
    if q_sqrt.get_shape().ndims==2:
AttributeError: 'numpy.ndarray' object has no attribute 'get_shape'
```
Here is some code to reproduce the error:
```
import GPflow
import numpy as np
X = np.reshape(np.linspace(0, 1, 100), (100,1))
Y = np.reshape(np.linspace(0, 1, 100), (100,1))
Z = Y[:10]
m = SVGP(X, Y, GPflow.kernels.RBF(1), GPflow.likelihoods.Gaussian(), Z)
m.q_sqrt.fixed = True
m._compile()
```

**Comment by jameshensman** [comment]
Small edit for reporducibility:
```python
import GPflow
import numpy as np
X = np.reshape(np.linspace(0, 1, 100), (100,1))
Y = np.reshape(np.linspace(0, 1, 100), (100,1))
Z = Y[:10]
m = GPflow.svgp.SVGP(X, Y, GPflow.kernels.RBF(1), GPflow.likelihoods.Gaussian(), Z)
m.q_sqrt.fixed = True
m._compile()
```

**Comment by jameshensman** [comment]
Thanks for the report @msbauer , this is definitely a bug,and I think it's easily fixed, I'll get right on it.

**Comment by jameshensman** [comment]
PR created to fix this, #47

**Comment by alexggmatthews** [comment]
Merged #47 so this is now fixed on master
